### PR TITLE
Remove the plain sql dependency.

### PR DESCRIPTION
**Pull request by ylamgarchal**
This patch removes the sql files and use only
SqlAlchemy core.

Change-Id: I56a241ba5b0fb742714bc076194667c0dc0a2650

### CODE REVIEW

File: server/tests/db_provision_test.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import bcrypt
+
+
+def provision(db_conn):
+
+    # Create teams
+    query = models_core.TEAMS.insert().values(name='admin')
+    db_conn.execute(query).inserted_primary_key[0]
+
```
Comment (IxDay) [inline server/tests/db_provision_test.py]: Maybe just for readability create a small helper which will avoid creating the query intermediate variable and directly return the primary key (will divide lines by two here)

something like

    def insert(query):
        return db_conn.execute(query).inserted_primary_key[0]

and use like this 

    company_a_id = insert(models_core.TEAMS.insert().values(name='company_a'))

**Comment by IxDay** [comment]
Just tell me what do you think about my comment and you can merge, it is ok for me. Good job !

**Comment by ylamgarchal** [comment]
I updated the PR. Is it good now ?

**Comment by IxDay** [comment]
Merged

**Pull request merged**